packages/utils/src/lib/reports/utils.unit.test.ts

```diff
@@ -594,3 +599,67 @@ describe('sortAuditIssues', () => {
     ]);
   });
 });
+
+describe('scoreMarker', () => {
+  const {
+    red: redCircle,
+    yellow: yellowCircle,
+    green: greenCircle,
+  } = MARKERS.circle;
+  const {
+    red: redSquare,
+    yellow: yellowSquare,
+    green: greenSquare,
+  } = MARKERS.square;
+
+  it('should return circle by default', () => {
+    expect(scoreMarker(0)).toBe(redCircle);
+  });
+
+  it.each<[string, MarkerShape | undefined]>([
+    [redCircle, undefined],
+    [redCircle, 'circle'],
+    [redSquare, 'square'],
+  ])('should return icon %s for marker type %s', (icon, type) => {
+    expect(scoreMarker(0, type)).toBe(icon);
+  });
+
+  it.each<[string, number]>([
+    [redCircle, 0],
+    [redCircle, SCORE_COLOR_RANGE.YELLOW_MIN - 0.001],
+    [yellowCircle, SCORE_COLOR_RANGE.YELLOW_MIN],
+    [yellowCircle, SCORE_COLOR_RANGE.GREEN_MIN - 0.001],
+    [greenCircle, SCORE_COLOR_RANGE.GREEN_MIN],
+    [greenCircle, 1],
+  ])(
+    'should return circle icon %s for score %s if type is circle',
+    (icon, score) => {
+      expect(scoreMarker(score, 'circle')).toBe(icon);
+    },
+  );
+
+  it.each<[string, number]>([
+    [redSquare, 0],
+    [redSquare, SCORE_COLOR_RANGE.YELLOW_MIN - 0.001],
+    [yellowSquare, SCORE_COLOR_RANGE.YELLOW_MIN],
+    [yellowSquare, SCORE_COLOR_RANGE.GREEN_MIN - 0.001],
+    [greenSquare, SCORE_COLOR_RANGE.GREEN_MIN],
+    [greenSquare, 1],
+  ])(
+    'should return circle icon %s for score %s if type is square',
+    (icon, score) => {
+      expect(scoreMarker(score, 'square')).toBe(icon);
+    },
+  );
+});
+
+describe('getSeverityIcon', () => {
```

Comment:
Function was renamed.

```suggestion
describe('severityMarker', () => {
```

Reply:
changed